plugin/src/main/java/com/iridium/iridiumskyblock/gui/BankGUI.java

```diff
@@ -56,6 +56,7 @@ public void addContent(Inventory inventory) {
             IslandBank islandBank = IridiumSkyblock.getInstance().getIslandManager().getIslandBank(island, bankItem);
             inventory.setItem(bankItem.getItem().slot, ItemStackUtils.makeItem(bankItem.getItem(), Collections.singletonList(new Placeholder("amount", bankItem.toString(islandBank.getNumber())))));
         }
+        inventory.setItem(IridiumSkyblock.getInstance().getInventories().bankTransactions.slot, ItemStackUtils.makeTransaction(IridiumSkyblock.getInstance().getInventories().bankTransactions, island));
```

Comment:
From what I can see this itemstack does nothing?

Reply:
Nevermind I understand now, but if you withdraw 0 money it shouldnt add as a log (e.g. if you spam click the item when you have no money in there)

Reply:
them adding on Deposit and Withdraw commands, so command is runs succes will add it otherwise not